src/pages/SignUp/services/SignUp.Hooks.js

```diff
@@ -6,22 +6,26 @@ import { paths } from '../../../services/path';
 
 export const useSignUp = () => {
   const [loading, setLoading] = useState(false);
+  const [navbar,setNavbar] = useState(false);
+
   const toast = useToast();
   const navigate = useNavigate();
 
   return {
     signUp: async ({ email, password }) => {
       try {
         setLoading(true);
+        setNavbar(true);
+
 
-        const { error } = await supabase.auth.signUp({ email, password });
+        const { error } = await supabase.auth.signUp({ email, password, navbar });
         if (error) throw error;
         toast({
           status: 'success',
           title: 'Sign up success',
           description: 'Check your mailbox!',
         });
-        navigate(paths.signin);
+        navigate(paths.home);
```

Comment:
These are not in the scope of the task. Please revert them back.

Reply:
I changed it

Reply:
You haven't pushed the changes here. If you've done it, it shouldn't show on this PR.